regression/contracts/invar_check_pointer_modifies/test.desc

```diff
@@ -0,0 +1,14 @@
+CORE
+main.c
+--enforce-all-contracts --pointer-check
+^EXIT=0$
+^SIGNAL=0$
+^\[main.1\] .* Check loop invariant before entry: SUCCESS$
+^\[main.2\] .* Check that loop invariant is preserved: SUCCESS$
+^\[main.assertion.1\] .* assertion r == 0: SUCCESS$
+^VERIFICATION SUCCESSFUL$
```

Comment:
Can we also add a negative test?

Reply:
Good idea! Added one for aliased pointers :)